framework/db/pgsql/Schema.php

```diff
@@ -142,12 +142,13 @@ protected function resolveTableName($name)
      */
     protected function findSchemaNames()
     {
-        $sql = <<<'SQL'
-SELECT ns.nspname AS schema_name
-FROM pg_namespace ns
-WHERE ns.nspname != 'information_schema' AND ns.nspname NOT LIKE 'pg_%'
-ORDER BY ns.nspname
+        static $sql = <<<'SQL'
```

Comment:
Doesn't seem to be MSSQL-related.

Reply:
Just a findSchemaNames normalization. This PR handles all of supported DBMSes.

Reply:
Then the changelog should be adjusted.

Reply:
No need to strip anything from the PR.

Reply:
Adjusted. I don't know, maybe "Fixed `yii\db\Schema::getSchemaNames()` for MSSQL, updated SQL queries and added tests for all DBMSes" sounds even better.